packages/blitz/src/render.rs

```diff
@@ -237,24 +280,42 @@ where
     }
 
     pub fn click(&mut self) {
+        let Some(node_id) = self.hover_node_id else {
+            return;
+        };
+
+        let RenderState::Active(state) = &self.render_state else {
+            return;
+        };
+
         if self.devtools.highlight_hover {
-            if let Some(node_id) = self.hover_node_id {
-                let node = &self.dom.as_ref().tree()[node_id];
-                println!("Node {} {}", node.id, node.node_debug_str());
-                dbg!(&node.final_layout);
-                dbg!(&node.style);
-
-                let children: Vec<_> = node
-                    .children
-                    .iter()
-                    .map(|id| &self.dom.as_ref().tree()[*id])
-                    .map(|node| (node.id, node.order(), node.node_debug_str()))
-                    .collect();
-
-                println!("Children: {:?}", children);
-                // taffy::print_tree(&self.dom, node_id.into());
-            }
+            let node = &self.dom.as_ref().tree()[node_id];
+            println!("Node {} {}", node.id, node.node_debug_str());
+            dbg!(&node.final_layout);
+            dbg!(&node.style);
+
+            let children: Vec<_> = node
+                .children
+                .iter()
+                .map(|id| &self.dom.as_ref().tree()[*id])
+                .map(|node| (node.id, node.order(), node.node_debug_str()))
+                .collect();
+
+            println!("Children: {:?}", children);
+            // taffy::print_tree(&self.dom, node_id.into());
         }
+
+        // If we hit a node, then we collect the node to its parents, check for listeners, and then
+        // call those listeners
+
+        self.dom.handle_event(RendererEvent {
+            name: "click".to_string(),
+            target: node_id,
+            data: EventData::Click {
+                x: self.mouse_pos.0 as f64,
+                y: self.mouse_pos.1 as f64,
+            },
+        });
```

Comment:
This should probably be in an `else`. Devtools inspector being enabled usually disables the app from handling clicks (clicks highlight element in devtools instead).